src/runtime_src/tools/scripts/apu_recipes/skd.service

```diff
@@ -1,10 +1,8 @@
 [Unit]
 Description=Soft Kernel Daemon
-After=apu-boot.service
 
 [Service]
 Type=forking
-User=softkernel
```

Comment:
why did we remove the user?
@jeffli-xilinx , please review this change

Reply:
@rbramand-xilinx 
Please keep User=softkernel

You can modify apu-boot and move all the lines that come after rpm-channel/ready to before the if statement.

Original:
if [ -e /sys/bus/platform/devices/rpu-channel/ready ]; then
	echo 1 > /sys/bus/platform/devices/rpu-channel/ready
	# Add softkernel user for PS kernel daemon to run as
	useradd softkernel
	# Work-around for AIE as it defaults root access only
	chmod 666 /dev/aie0
	# Create default xrt.ini
	echo "# xrt.ini generated by apu-boot script" > /usr/bin/xrt.ini
	echo "[Runtime]" >> /usr/bin/xrt.ini
	echo "verbosity=5" >> /usr/bin/xrt.ini
	echo "runtime_log=syslog" >> /usr/bin/xrt.ini
fi

Update:
# Add softkernel user for PS kernel daemon to run as
useradd softkernel
# Work-around for AIE as it defaults root access only
chmod 666 /dev/aie0
# Create default xrt.ini
echo "# xrt.ini generated by apu-boot script" > /usr/bin/xrt.ini
echo "[Runtime]" >> /usr/bin/xrt.ini
echo "verbosity=5" >> /usr/bin/xrt.ini
echo "runtime_log=syslog" >> /usr/bin/xrt.ini
if [ -e /sys/bus/platform/devices/rpu-channel/ready ]; then
	echo 1 > /sys/bus/platform/devices/rpu-channel/ready
fi